source/common/http/http1/conn_pool.cc

```diff
@@ -74,16 +74,16 @@ void ActiveClient::StreamWrapper::onResetStream(StreamResetReason, absl::string_
 ActiveClient::ActiveClient(HttpConnPoolImplBase& parent)
     : Envoy::Http::ActiveClient(
           parent, parent.host()->cluster().maxRequestsPerConnection(),
-          1 // HTTP1 always has a concurrent-request-limit of 1 per connection.
-      ) {
+          1, // HTTP1 always has a concurrent-request-limit of 1 per connection.
+          1, absl::nullopt) {
```

Comment:
nit would you mind adding comments (or /*effective_concurrent_stream_limit=*/, etc) to the literals?